tests/cover/test_conjecture_engine.py

```diff
@@ -759,7 +761,7 @@ def f(data):
 
     runner = ConjectureRunner(f)
     runner.run()
-    assert len(runner.target_selector.examples_by_tags) == 3
+    assert len(runner.target_selector.examples_by_tags) == 7
```

Comment:
Not sure if I should do something better here than just updating this number...

Reply:
Hmm. It's not a great test. 

Possibly the thing to do would be to update it to assert that the tags in `examples_by_tag` are either `universal` or a `Negated` of some other tag that is also in the dict?